### PR TITLE
chore(feed): Remove deprecated Stylesheet.create usages

### PR DESCRIPTION
**Pull request by WaDadidou**
I removed some deprecated `StyleSheet` usages

### CODE REVIEW

**Review by n0izn0iz — DISMISSED** [review]
LGTM

**Review by n0izn0iz — APPROVED** [review]
LGTM

File: packages/components/ScreenContainer/index.tsx [hunk 1]
```diff
@@ -126,7 +125,14 @@ export const ScreenContainer: React.FC<ScreenContainerProps> = ({
     <SafeAreaView style={{ width: "100%", flex: 1 }}>
       {/*FIXME: Too many containers levels*/}
 
-      <View style={styles.container}>
+      <View
+        style={{
+          flex: 1,
+          backgroundColor: "#000000",
```
Comment (hthieu1110) [inline packages/components/ScreenContainer/index.tsx:131]: should we use `secondaryColor` instead of hardcoded one ?

Comment (n0izn0iz) [reply packages/components/ScreenContainer/index.tsx:131]: I think this PR is not the place to edit the styles

**Review by hthieu1110 — APPROVED** [review]
LGTM

**Pull request merged**